string name
uint8  id
int32  current_position
int32  current_velocity
int16  current_load